<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="sayLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="sayButton">
      <property name="text">
       <string>전송</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="roomTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>48</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_LIG">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Checkbox_Off.png</normaloff>
     <normalon>:/image/Checkbox_On.png</normalon>
     <disabledoff>:/image/Checkbox_Off.png</disabledoff>
     <disabledon>:/image/Checkbox_On.png</disabledon>
     <activeoff>:/image/Checkbox_Off.png</activeoff>
     <activeon>:/image/Checkbox_On.png</activeon>
     <selectedoff>:/image/Checkbox_Off.png</selectedoff>
     <selectedon>:/image/Checkbox_On.png</selectedon>:/image/Checkbox_Off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QLabel" name="label_Humi_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>내부 습도값</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLCDNumber" name="lcdNumber_Temp_1"/>
    </item>
    <item row="2" column="1">
     <widget class="QLCDNumber" name="lcdNumber_Humi_1"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Temp_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>내부 온도값</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_WIN">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Checkbox_Off.png</normaloff>
     <normalon>:/image/Checkbox_On.png</normalon>
     <disabledoff>:/image/Checkbox_Off.png</disabledoff>
     <disabledon>:/image/Checkbox_On.png</disabledon>
     <activeoff>:/image/Checkbox_Off.png</activeoff>
     <activeon>:/image/Checkbox_On.png</activeon>
     <selectedoff>:/image/Checkbox_Off.png</selectedoff>
     <selectedon>:/image/Checkbox_On.png</selectedon>:/image/Checkbox_Off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/image/2554083.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/image/4388775.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/image/2688820.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_WAT">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Checkbox_Off.png</normaloff>
     <normalon>:/image/Checkbox_On.png</normalon>
     <disabledoff>:/image/Checkbox_Off.png</disabledoff>
     <disabledon>:/image/Checkbox_On.png</disabledon>
     <activeoff>:/image/Checkbox_Off.png</activeoff>
     <activeon>:/image/Checkbox_On.png</activeon>
     <selectedoff>:/image/Checkbox_Off.png</selectedoff>
     <selectedon>:/image/Checkbox_On.png</selectedon>:/image/Checkbox_Off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/image/temperature.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/image/humidity.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
